Application/EdFi.Ods.Features/Controllers/OpenApiMetadataController.cs

```diff
@@ -115,7 +115,7 @@ string GetMetadataUrlSegmentForCacheItem(OpenApiContent apiContent, int? schoolY
                     ? string.Empty
                     : apiContent.RelativeSectionPath.EnsureSuffixApplied("/");
 
-                return $"{basePath}/{relativeSectionUri}{SwaggerDocumentHelper.SwaggerJson}".ToLowerInvariant();
+                return $"{basePath}/{relativeSectionUri}{OpenApiMetadataDocumentHelper.SwaggerJson}".ToLowerInvariant();
```

Comment:
```suggestion
                return $"{basePath}/{relativeSectionUri}{OpenApiMetadataDocumentHelper.Json}".ToLowerInvariant();
```

Reply:
make sure you rename using resharper to get all the changes

Reply:
Fixed